.eslintrc.json

```diff
@@ -0,0 +1,60 @@
+{
+    "plugins": [
+      "prettier"
+    ],
+    "rules": {
+      "prettier/prettier": [
+        "error",
+        {
+          "singleQuote": true
+        }
+      ]
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:prettier/recommended"
+    ],
+    "env": {
+      "es6": true,
+      "jest": true,
+      "node": true,
+      "browser": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module"
+    },
+    "overrides": [{
+      "files": ["packages/**/*.js"],
+      "excludedFiles": ["*.test.js","packages/**/*.test.js"],
+      "plugins": [
+        "prettier",
+        "es5"
+      ],
+      "env": {
+        "browser": true
```

Comment:
If we remove this, does it break? (there's an environment `browser: true` above)

Reply:
If we remove it, then it will throw a bunch of errors saying that `document` and `window` variable aren't defined. But once you tell it the environment is a browser, it realizes that `document` and `window` are going to be defined as "browser global variables"; source: https://eslint.org/docs/user-guide/configuring#specifying-environments.

Reply:
Thanks for the link and explanation!